README.md

```diff
@@ -1070,8 +1079,10 @@ Front. Neurosci. (2016), 10:496. [Dataset](https://github.com/qian-liu/benchmark
 IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2019. [Project page](https://spritea.github.io/DET/).
 - <a name="Miao19fnbot"></a>Miao, S., Chen, G., Ning, X., Zi, Y., Ren, K., Bing, Z., Knoll, A.,  
 *[Neuromorphic Vision Datasets for Pedestrian Detection, Action Recognition, and Fall Detection](https://doi.org/10.3389/fnbot.2019.00038),*  
-Front. Neurorobot. (2019). [Dataset](https://github.com/MSZTY/PAFBenchmark)
-
+Front. Neurorobot. (2019). [Dataset](https://github.com/MSZTY/PAFBenchmark) 
+ - <a name="bi2019graph"></a>Yin Bi, Aaron Chadha, Alhabib Abbas, Eirina Bourtsoulatze, Yiannis Andreopoulos,  
```

Comment:
Here, there is no need to put an anchor/key again, just a link to the previous anchor. It avoids duplicates.
See other examples in the readme, such as the entry from Bardow et al in the section on image intensity reconstruction.